### DISCUSSION TITLE
デフォルトRunbookにある`Resolve`ボタンが存在しないようにみえる

### DISCUSSION THREAD

**Discussion by yuuki** [topotal/waroom-roadmap#14] [Others]
デフォルトRunbookの文言のうち、Resolution 2/2「パーティションの確認」について、

```
問題なくサービスが稼働していることを確認でき次第、下の Resolve ボタンを押して Resolve ステータスに変更してください。
```

とあります。しかし、インシデント詳細ページの下には、`Acknowledge incident`ボタンはあっても`Resolve`ボタンがないことに気づきました。その後、 `Acknowledge incident`ボタンを押すと、`Resolve incident`ボタンが出てきました。
おそらくは、Resolutionプロセスに入るどこかの時点で、`Acknowledge incident`ボタンを押すことを想定されているのだと思いますが、`Acknowledge incident`ボタンをどのタイミングで押すのか、またこのボタンにはどのような意図があるのかわかりにくく感じました。

**Comment by sawa-zen** [comment]
@yuuki 
たしかに `Acknowledge`  を促すステップをデフォルトの Runbook にいれるか、一つでも Runbook のステップを実行した時点で `Acknowledge` のステータスに自動的に変更されるかのいずれかであってほしいですね 🤔 ありがとうございます 👍🏻
